### DISCUSSION TITLE
Safari extension

### DISCUSSION THREAD

**Discussion by pindab0ter** [tamius-han/ultrawidify#218] [Ideas]
This extension is amazing, so first of all, thanks to everyone contributing!

I would love it if there was a Safari extension as well. Is this possible and/or doable?

**Comment by tamius-han** [comment]
It is technically possible, but it's not going to happen for the foreseeable future. 

Main reasons:

* You pretty much need to have apple hardware (or try your luck with a hackintosh, which Apple isn't a big fan of)
* Looks like you need to be enrolled into Apple Developer Program, which is like €100 a year last time I checked. And I really don't feel like paying that kind of money for the privilege.

For comparison — Firefox is free. Microsoft is free. Chrome Web Store was €5 one-time fee. And while I _could_ look into charging some "give me a beer a month" tier subscription money for Safari version in order to offset the cost ... 1. how many people would pay for it, when Chrome version is free (and/or when free alternatives prolly exist) and 2. once people are required to pony up some cash, the expectations go _way_ up. A bit above the current state of the extension.

**Reply by pindab0ter** [reply]
Thank you for your quick reply! I'm sorry to hear that this isn't going to be a thing in the foreseeable future. For what it's worth, I'd gladly pay a buck or two for an extension like this. This is in line with what extensions like SponsorBlock cost, too.

For now, I'll just have to use Firefox if I want to watch a widescreen video. Again, thanks for all you've done!

**Comment by bystritskiy** [comment]
I looked all over the Internet and did not find an analogue of such an extension for safari. I only found about a dozen topics from reddit and forums and etc where people ask for such an extension. Absolutely free niche. So it could be a good investment! :) @pindab0ter
